### PR TITLE
Carp classic carpotoxin fix

### LINKED ISSUE #49588: Carp classic cigarettes do not contain carpotoxin
Issue reported from  Round ID: 130961 (/tg/Station Event Hall [ENGLISH] [US-West] [100% FREE LAG])

Reporting client version: 513.1511

Ran a local with the intent of mixing up some zombie powder, was checking cigarettes from every vending machine I could find with a signal ace 2 cartridge. The results from scanning said none of them contained the toxin, so I ground them all to verify, and did not receive any carpotoxin. Tested 132 cigarettes in this experiment.

114 cigs

For the sake of being thorough I then hopped on sybil-2 to verify, and went through an additional 144 cigarettes, this time only checking them with the signal ace 2 cartridge, as I wasn't about to grind so many goddamn cigarettes again. 

Not a single unit of carpotoxin in 276 attempts at a 5%? I can't believe I'm that unlucky.

### PR DESCRIPTION
**Pull request by ArcaneDefence**
##  About The Pull Request
Fixes https://github.com/tgstation/tgstation/issues/49588

## Why It's Good For The Game

Makes the carp classic cigarettes work as intended

## Changelog
:cl:
fix: The recipe for Carp Classic cigarettes has been brought up to company standards.
/:cl:

### CODE REVIEW

**Comment by GunHog** [comment]
Good first PR!

**Comment by LemonInTheDark** [comment]
Good work. The moonrunes are clearing.

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -330,9 +330,9 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	desc = "A Carp Classic brand cigarette. A small label on its side indicates that it does NOT contain carpotoxin."
 
 /obj/item/clothing/mask/cigarette/carp/Initialize()
-	. = ..()
 	if(prob(5))
 		list_reagents = list(/datum/reagent/drug/nicotine = 15, /datum/reagent/toxin/carpotoxin = 3) //they lied
+	. = ..()
```
Comment (ShizCalev) [inline code/game/objects/items/cigs_lighters.dm]: return ..()

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Review by ExcessiveUseOfCobblestone — COMMENTED** [review]
it's just adding the carpotoxin to the default value of nicotine so probably safer to not edit the list at all and just add the desired reagent on top.  That way if default content changes we don't have to maintain the second list in initialize as well.

```DM
/obj/item/clothing/mask/cigarette/carp/Initialize()
    . = ..()
    if(!prob(5))
        return
    reagents?.add_reagent(/datum/reagent/toxin/carpotoxin , 3) // They lied

File: code/game/objects/items/cigs_lighters.dm (hunk 1 above)
Comment (ArcaneDefence) [reply code/game/objects/items/cigs_lighters.dm]: Went with Cobby's suggestions as they make sense to me.

**Pull request merged**